### PR TITLE
[#220] add last move display

### PR DESCRIPTION
**Pull request by benshumaker**
Closes: #220

Also, I realized that move_history is perfect for the soundEffects. I reworked the audio a bit. gameSubscription no longer plays SFX.

### CODE REVIEW

File: frontend/packages/web/src/components/features/BoardDisplay/Square.tsx [hunk 1]
```diff
@@ -50,6 +50,18 @@ export default function Square({ children, row, col, color }: any) {
     return 'none';
   };
 
+  const getColor = () => {
+    if (!game?.board_state?.move_history || game?.board_state?.move_history.length < 1)
+      return color;
+    const previous_move = game.board_state.move_history.slice(-1)[0];
+    if (previous_move.destination.rank == row && previous_move.destination.file == col) {
+      return color === '#ffffff' ? '#faafaa' : '#9e564c';
```
Comment (izakfr) [inline frontend/packages/web/src/components/features/BoardDisplay/Square.tsx:58]: Can you alias these strings with consts at the top of the file?

File: frontend/packages/web/src/components/features/SoundEffects/SoundEffects.tsx [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+import React, { useEffect } from 'react';
 import moveSound from 'assets/moveSound.mp3';
 import checkSound from 'assets/checkSound.mp3';
+import { useRecoilValue } from 'recoil';
+import { gameState } from 'store';
 
 export const SoundEffects = () => {
+  const game = useRecoilValue(gameState);
+
+  useEffect(() => {
```
Comment (izakfr) [inline frontend/packages/web/src/components/features/SoundEffects/SoundEffects.tsx:10]: nice

**Review by izakfr — CHANGES_REQUESTED** [review]

File: frontend/packages/web/src/components/features/BoardDisplay/Square.tsx (hunk 1 above)
Comment (benshumaker) [reply frontend/packages/web/src/components/features/BoardDisplay/Square.tsx]: Alright, now we have a colors asset.

**Review by izakfr — APPROVED** [review]
Much better styling, thanks!

**Pull request merged**